src/ResourceManager/Compute/Commands.Compute/packages.config

```diff
@@ -3,7 +3,10 @@
   <package id="AutoMapper" version="3.1.1" targetFramework="net45" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="16.2.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="12.0.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="14.0.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="4.1.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.9" targetFramework="net452" />
```

Comment:
@EvgenyAgafonchikov same comment, you shouldn't need to add the references in this file, they should be derived from the [`packages.config`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ResourceManager.Common/packages.config) file in `Commands.ResourceManager.Common`